crates/swc_cli/Cargo.toml

```diff
@@ -12,6 +12,14 @@ version = "0.26.1"
 name = "swc"
 path = "./src/main.rs"
 
+[features]
+default = ["plugin"]
+plugin = [
+  "swc/plugin",
+  "wasmer/default",
```

Comment:
This will break `wasm/plugin`

Reply:
It should be `wasm/plugin` => `swc/plugin` && `wasmer/js-default`

Reply:
Swc cli shouldn't enable wasmer/js, isn't it?

Reply:
Oh, sorry. this comment is my mistake.